src/LGPL-2.0-only.xml

```diff
@@ -0,0 +1,635 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+  <license licenseId="LGPL-2.0-only" isOsiApproved="true"
+  name="GNU Library General Public License v2 only">
+    <crossRefs>
+      <crossRef>http://www.gnu.org/licenses/old-licenses/lgpl-2.0-standalone.html</crossRef>
+    </crossRefs>
+    <standardLicenseHeader>
+      Copyright (C)<alt name="copyright" match=".+">year name of author</alt>
+      This library is free software; you can redistribute it and/or modify it
+      under the terms of the GNU Library General Public License as published
+      by the Free Software Foundation; version 2. This library is distributed
+      in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+      even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+      PURPOSE. See the GNU Library General Public License for more details.
+      You should have received a copy of the GNU Library General Public
+      License along with this library; if not, write to the Free Software
+      Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
+    </standardLicenseHeader>
+    <notes>
+      This license was released: June 1991. This license has
+      been superseded by LGPL v2.1 This refers to when this
+      LGPL 2.0 only is being used (as opposed to "or later).
```

Comment:
Missing end " after `later`

Reply:
Note - this was an issue with the original text unrelated to the changes for the new license ID's